caiman/cluster.py

```diff
@@ -406,7 +406,8 @@ def setup_cluster(backend='multiprocessing', n_processes=None, single_thread=Fal
                 raise Exception(
                     'A cluster is already runnning. Terminate with dview.terminate() if you want to restart.')
             c = None
-            dview = Pool(n_processes)
+            ctx = multiprocessing.get_context('forkserver')
```

Comment:
Does this lock us into forkserver for these two backends? If so, was this intended?

Reply:
It makes forkserver the start method, instead of 'fork', for the multiprocessing backend only.   According to scipy apple believes that 'fork' is buggy and should not be used.
Perhaps this should be only changed for Mac.